.circleci/build-openapi.sh

```diff
@@ -9,5 +9,25 @@ FNAMES=`find . -name "openapi.yaml" -not -path "./fragments/*" -not -path "./nod
 for fin in $FNAMES; do
     fout=./build/$fin
     mkdir -p ${fout%/*}
-    speccy resolve $fin > $fout
-done
\ No newline at end of file
+    #speccy resolve $fin > $fout
+    swagger-cli bundle $fin -o $fout -t yaml
+    cp build/index.html ${fout%/*}/
+done
+
+cp build/core/openapi.yaml build/openapi.yaml
```

Comment:
Do you also need to copy the html? It may exist at the moment in dev, but I guess not in the new folder for the tag?

Reply:
By the way, I think dev should be "cleaned" before each CI run to ensure we don't have "old" files floating around.

Reply:
No, the one in build/ already is actually the committed copy that gets used to create the others

Reply:
So it will be under any new tag as well

Reply:
Oh, great!

Reply:
@m-mohr WRT to cleaning the dev folder, the gh-pages plugin apparently does a force push for everything in that repo from that directory down, so everything under the dev branch is completely replaced every time.